tests/archiver.rs

```diff
@@ -580,15 +598,16 @@ impl Override {
     }
 }
 
-const OVERRIDES: [Override; 19] = [
+const OVERRIDES: [Override; 20] = [
     Override::alias("apa", "american-psychological-association", &["apa"]),
     Override::alias("bmj", "british-medical-journal", &["bmj"]),
     Override::first(
         "china-national-standard-gb-t-7714-2015-author-date",
         "gb-7714-2015-author-date",
     ),
-    Override::first("chicago-fullnote-bibliography", "chicago-fullnotes"),
-    Override::first("chicago-note-bibliography", "chicago-notes"),
+    Override::first("chicago-notes-bibliography", "chicago-notes"),
+    Override::first("chicago-shortened-notes-bibliography", "chicago-fullnotes"),
```

Comment:
This doesn't seem right. `chicago-fullnotes` should alias `chicago-notes`. I feel like you will need to use `Override::alias` for this like in the apa example.

In addition, shortened-notes should be available on its own. (Fixing this line should be enough.)